core/client/fs/src/test/java/alluxio/client/file/FileSystemContextTest.java

```diff
@@ -93,4 +98,75 @@ public void run() {
       mFsCtx.releaseMasterClient(client);
     }
   }
+
+  @Test
+  public void excessWorkerGroupTest() throws Exception {
```

Comment:
what is this testing exactly?

Reply:
I think I committed this by accident....I don't even know how this got in here